### ISSUE TITLE
[Bug]: Token image rendered smaller in initiative panel when states are toggled off

### ISSUE THREAD

**Issue by kwvanderlinde** [RPTools/maptool#4984]
### Describe the Bug

Using the initiative panel with "Show Token States" enabled will render the token at a size comparable to the initiative label itself. Toggling the option off makes the token image about half the size.

### To Reproduce

1. Toggle "Show Token States" on.
2.  Add a circle token to initiative
3. Toggle "Show Token States" off.

### Expected Behaviour

The token image should remain at the larger size when "Show Token States" is toggled off.

### Screenshots

With "Show Token States" on:
![image](https://github.com/user-attachments/assets/84c776a7-4a44-48c4-8efc-47e393c7ce70)

With it off:
![image](https://github.com/user-attachments/assets/0e57ba1d-092a-4604-b776-3721424e9afb)

### MapTool Info

1.15.2

### Desktop

Linux Mint 22

**Comment by kwvanderlinde** [comment]
Took a gander at the code, looks like this behaviour was deliberate. Still, I don't get the point of it and was pretty surprised as a user!

**Comment by FullBleed** [comment]
It seem like the states should not affect the size.  And if the size is going to change there should be an explicit "compact" option (maybe only available with the states off.)

**Comment by bubblobill** [comment]
I expect many states are impossible to see on the itty-bitty version

**Comment by FullBleed** [comment]
> I expect many states are impossible to see on the itty-bitty version

True.  But it still seems strange that if you're looking to change the display size it's controlled by whether you display states.

It would actually be nice to have more than two display sizes for higher resolution screens.  The init panel is currently kind of useless at 4k.
